docs/getting-started.md

```diff
@@ -67,16 +67,21 @@ Verder zijn er nog een paar algemene functies die gelden voor alle bovenstaande
 ## Probeer en test de API
 De werking van de API is het makkelijkst te testen met behulp van [Postman](https://www.getpostman.com/).
 De [openapi.yaml](../specificatie/BRK-Bevragen/genereervariant/openapi.yaml) kun je importeren als project, waarna de verschillende requests worden ingeladen die deze API ondersteunt.
-We hebben al een project voor je gemaakt die je kan gebruiken: [BRK-Bevragen-postman-collection.json](../test/BRK-Bevragen-postman-collection.json). Hierin moet je alleen de endpoints en authenticatie (API key) nog invullen.
+We hebben al een project voor je gemaakt die je kan gebruiken: [BRK-Bevragen-postman-collection.json](../test/BRK-Bevragen-postman-collection.json). Hierin moet je alleen de endpoints en authenticatie (API-key) nog invullen.
 
-### URL
-De testomgeving van de API is te benaderen via de volgende url: https://api.test.kadaster.nl/esd/gemeenten/brk/.
+Voor de connectie met de testomgeving van deze API is vereist:
+- Internet toegang tot het Kadaster endpoint via IPv4 of IPv6 en met het TLS 1.2 protocol.
+- Een Staat der Nederlanden root CA - G3 certificaat in de truststore, zie hiervoor: https://www.pkioverheid.nl/
```

Comment:
Betekent dit dat iemand een PKI overheid certificaat op zijn PC of laptop moet installeren voordat die met Postman de API kan gaan uitproberen?

Kan je de toegang tot de testomgeving niet laagdrempeliger houden?

Reply:
Ik ga dit na bij Rene.